php/Terminus/AnnotatedCommands/ExampleCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Terminus\AnnotatedCommands;
+
+use Robo\ResultData;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputAwareInterface;
+use Robo\Contract\OutputAwareInterface;
+use Robo\Common\OutputAwareTrait;
+use Robo\Common\InputAwareTrait;
+
+// TODO: add input and output aware interfaces
+class ExampleCommand implements InputAwareInterface, OutputAwareInterface
```

Comment:
Typical Robo commands will extend Robo\Tasks, in order to use the various Robo task methods. This may also be useful in Terminus, but is not needed.  A Robo command file does not need to extend any class or implement any interfaces.  Here, InputAwareInterface and OutputAwareInterface are implemented in order to gain access to the Symfony Console InputInterface and OutputInterface objects for IO, via `input()` and `output()` methods, respectively.